eclass/bzr.eclass

```diff
@@ -287,3 +292,8 @@ bzr_fetch() {
 bzr_src_unpack() {
 	bzr_fetch
 }
+
+_BZR_ECLASS=1
+fi
+
+EXPORT_FUNCTIONS src_unpack
```

Comment:
Why that moving around of EXPORT_FUNCTIONS?
Please leave bzr.eclass alone altogether, I don't need any help maintaining it.